applications/subxt-python.md

```diff
@@ -0,0 +1,196 @@
+    - The Python wrapper complements, not replaces, the `subxt` library. Parity's `subxt` remains the primary and most
+    up-to-date source for Substrate interactions.
+    - Potential performance overhead compared to native `subxt`. Optimization might be a future goal but not an initial
+    priority.
+    - While the goal is to make the library friendly for Python developers, it might initially heavily reflect the
+    structure of the Rust `subxt` library. Additional Pythonic interfaces could be developed in later stages. 
+
+
+### Ecosystem Fit
+
+- Where and how does your project fit into the ecosystem?
+  - Serves as a bridge between the Python developer community and the Substrate ecosystem, allowing Python developers
+  to easily build and integrate applications with Substrate-based networks. `subxt-python` leverages the existing
+  `subxt` Rust library and provides a Python wrapper around it using PyO3. This allows Python developers to benefit
+  from the features and functionality of `subxt` while working in their preferred language.
+- Who is our target audience?
+  - Python developers interested in building applications that interact with Substrate-based blockchains.
+  - Developers working on parachains, dApps, wallets, or UI projects who want to use Python for their development stack.
+  - Researchers and data analysts who want to query and analyze data from Substrate-based networks using Python.
+  - Existing Python-based projects or organizations looking to integrate with Substrate/Polkadot/Kusama ecosystems.
+- What need(s) does the project meet?
+  - Provides a Python interface for interacting with Substrate-based blockchains, enabling Python developers to build
+  and integrate applications easily.
+  - Allows Python developers to leverage the extensive Python ecosystem and libraries for data analysis, machine
+  learning, and other use cases in conjunction with Substrate-based blockchains.
+  - Facilitates the adoption and growth of the Substrate/Polkadot/Kusama ecosystem by attracting Python developers and
+  expanding the developer base. Existing Substrate teams could leverage Python developers to build their projects. 
+- How did we identify these needs? Please provide evidence in the form of (scientific) articles, forum discussions, case studies, or raw data.
+  - An analysis of the Substrate/Polkadot/Kusama ecosystem reveals a lack of mature and actively maintained Python
+  libraries for interacting with Substrate-based blockchains, indicating a gap that `subxt-python` aims to fill.
+  - There's about 104 questions on [substrate stackexchange](https://substrate.stackexchange.com/search?q=python) 
+    signaling the need for Python support in the Substrate ecosystem.
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  - Yes, there is a project called [py-substrate-interface](https://github.com/polkascan/py-substrate-interface) which
```

Comment:
I still don't quite understand the advantage of building on top of subxt, could you elaborate? py-substrate-interface seems to be actively maintained.